src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView/bpp.jelly

```diff
@@ -365,19 +367,23 @@
 									<!-- Only adding the revision-cell once. -->
 									<j:if test="${x == 0}"><td class="revision-cell" rowspan="${buildGrid.rows}">
 										<table class="revision rounded build-card">
-											<tbody>
-												<tr class="header">
```

Comment:
A custom CSS theme has been a common approach to Jenkins customization for years. Why do you think that it is a bad practice? What is the good practice then?